### PR TITLE
[DOC] format docstrings in `feature_selection.py` correctly

### LINKED ISSUE #5993: [DOC] Bullet point not rendered properly on feature_selection.py
#### Describe the issue linked to the documentation

Under Time series transformations (https://www.sktime.net/en/latest/api_reference/transformations.html) in 
FeatureSelection , bullet point is not rendered properly for “feature-importances” under the  FeatureSelection class.

### PR DESCRIPTION
**Pull request by oleeviyababu**
The bullet point for the " feature-importance" is not properly rendered. I have added blank line before and after the bullet point list.

#### Reference Issues/PRs
Fixes #5993

#### What does this implement/fix? Explain your changes.
The bullet point for the " feature-importance" in feature_selection.py is not properly rendered. I have added blank line before and after the bullet point list.

#### Does your contribution introduce a new dependency? If yes, which one?

No

#### Any other comments?
No

##### For all contributions
- [ ] I've added myself to the [list of contributors](https://github.com/sktime/sktime/blob/main/CONTRIBUTORS.md) with any new badges I've earned :-)
  How to: add yourself to the [all-contributors file](https://github.com/sktime/sktime/blob/main/.all-contributorsrc) in the `sktime` root directory (not the `CONTRIBUTORS.md`). Common badges: `code` - fixing a bug, or adding code logic. `doc` - writing or improving documentation or docstrings. `bug` - reporting or diagnosing a bug (get this plus `code` if you also fixed the bug in the PR).`maintenance` - CI, test framework, release.
  See here for [full badge reference](https://allcontributors.org/docs/en/emoji-key)
- [ ] Optionally, I've added myself and possibly others to the [CODEOWNERS](https://github.com/sktime/sktime/blob/main/CODEOWNERS) file - do this if you want to become the owner or maintainer of an estimator you added.
  See here for further details on the [algorithm maintainer role](https://www.sktime.net/en/latest/get_involved/governance.html#algorithm-maintainers).
- [ ] The PR title starts with either [ENH], [MNT], [DOC], or [BUG]. [BUG] - bugfix, [MNT] - CI, test framework, [ENH] - adding or improving code, [DOC] - writing or improving documentation or docstrings.

##### For new estimators
- [ ] I've added the estimator to the API reference - in `docs/source/api_reference/taskname.rst`, follow the pattern.
- [ ] I've added one or more illustrative usage examples to the docstring, in a pydocstyle compliant `Examples` section.
- [ ] If the estimator relies on a soft dependency, I've set the `python_dependencies` tag and ensured
  dependency isolation, see the [estimator dependencies guide](https://www.sktime.net/en/latest/developer_guide/dependencies.html#adding-a-soft-dependency).

### CODE REVIEW

**Review by yarnabrina — APPROVED** [review]

**Comment by fkiraly** [comment]
FYI, your changes fail the code formatting checks, due to trailing whitespaces.

Here's a guide on how to set up automatic code formatting locally for you:
https://www.sktime.net/en/stable/developer_guide/coding_standards.html

**Comment by yarnabrina** [comment]
I didn't notice, not really readable from UI. Withdrawing approval, but this should be very simple to fix.

File: sktime/transformations/series/feature_selection.py [hunk 1]
```diff
@@ -25,13 +25,15 @@ class FeatureSelection(BaseTransformer):
     ----------
     method : str, required
         The method of how to select the features. Implemented methods are:
+
```
Comment (yarnabrina) [inline sktime/transformations/series/feature_selection.py:28]:
```suggestion

```

File: sktime/transformations/series/feature_selection.py [hunk 2]
```diff
@@ -25,13 +25,15 @@ class FeatureSelection(BaseTransformer):
...
         The method of how to select the features. Implemented methods are:
+        
         * "feature-importances": Use feature_importances_ of the regressor (meta-model)
           to select n_columns with highest importance values.
           Requires parameter n_columns.
         * "random": Randomly select n_columns features. Requires parameter n_columns.
         * "columns": Select features by given names.
         * "none": Remove all columns, transform returns None.
         * "all": Select all given features.
+
```
Comment (yarnabrina) [inline sktime/transformations/series/feature_selection.py:36]:
```suggestion

```

**Review by yarnabrina — CHANGES_REQUESTED** [review]

**Pull request merged**